src/docs/product/accounts/quotas/index.mdx

```diff
@@ -215,7 +215,7 @@ What this means is that if you experience a spike, we will temporarily protect y
 
 ### How Does Spike Protection Help?
 
-Because Sentry bills on monthly event volume, spikes can consume your Sentry capacity for the rest of the month. If it really is a spike, spike protection drops events during the spike to try and conserve your capacity. We also send an email notification to the Owner when spike protection is activated.
+Because Sentry bills on monthly event volume, spikes can consume your Sentry capacity for the rest of the month. If it really is a spike, spike protection drops events during the spike to try and conserve your capacity. We also send an email [notification](/product/alerts-notifications/notifications/#quota) to the Owner when spike protection is activated.
```

Comment:
```suggestion
Because Sentry bills on monthly event volume, spikes can consume your Sentry capacity for the rest of the month. If it really is a spike, spike protection drops events during the spike to try and conserve your capacity. We also send teammates with the "owner" organization permission level a [notification](/product/alerts-notifications/notifications/#quota) email.
```